corona-warn-app/cwa-app-android
# Local statistics connectivity (EXPOSUREAPP-8436)

This PR addresses Exposureapp-8436.
[Android] Statistics behavior when offline during onboarding.

How to test:
- visit statistic cards on the MainFragment
- check that its prevented to add a new card without internet connection (by notification or addView hiding)
- check if the addCard Element reappears after switching Internet/Wifi on
(if there is no cached data or internet connection, statistic data should not be shown)

Note: MERGE AFTER #3767 (done)

Screenshots:

No Internet Case:
- regular AddCard Item should not be visible (greyed out with explaining text)
![image](https://user-images.githubusercontent.com/46747780/127511230-6b2a6736-384b-4f47-b1e2-d14e41e6ee1c.png)



with internet:
![image](https://user-images.githubusercontent.com/46747780/127363148-a635a60d-b288-4e91-b5a7-ec45c8f2c463.png)
